### DISCUSSION TITLE
Tenants with and without subdomains, is it possible?

### DISCUSSION THREAD

**Discussion by alexv96** [hyn/multi-tenant#1014] [Q&A]
I have a platform that now we are going to make it pay now we have all our customers with their own subdomain, the idea now is to separate the payment platform with the free platform, ie that only the payment has subdomain while the free version can only enter the CENTRAL ROUTE, and that internally without placing in the URL the subdomain know which tenant corresponds.

Example
Central path: tenancy.test
Free version: No subdomain.
Paid version: They have their own subdomain.
Tenant1 (free), can only navigate to all routes but from tenancy.test/login
Tenant 2 (free), can only navigate to all routes but from tenancy.test/login
Tenant 3 (paid), can login from tenancy.test/login and only redirects you to your subdomain with your account which would be e.g. app3.tenancy.test/home

I do not know if you understand the idea, but is it possible to do that? and how could I do so that users do not have subdomain and navigate through the central path?

**Comment by fletch3555** [comment]
This would be infinitely easier with tenancy/tenancy instead of hyn/multi-tenant.

Having said that, I don't really have any guidance for you right now, so I'll leave this open for others to provide suggestions if they have any.
